developer-hub/src/meta_contributing.rs

```diff
@@ -0,0 +1,134 @@
+//! # Contribution
+//!
+//! The following sections cover more detailed information about this crate and how it should be
+//! maintained.
+//!
+//! ## Why Rust Docs?
+//!
+//! We acknowledge that blockchain based systems, particularly a cutting-edge one like Polkadot SDK
+//! is a software artifact that is complex, and rapidly evolving. This makes the task of documenting
+//! it externally extremely difficult, especially with regards to making sure it is up-to-date.
+//!
+//! Consequently, we argue that the best hedge against this is to move as much of the documentation
+//! near the source code as possible. This would further incentivize developers to keep the
```

Comment:
While I don't see any problems with this, would the only blocker of this being outside of polkadot_sdk be just docify not being able to retrieve code examples from remote repos?

Reply:
Yeah truthfully once https://github.com/sam0x17/docify/issues/14is resolved, 50% of this issue is solved. I think that is a big unlocker for you and nearby projects. Nonetheless, developers will prefer not needing to update code and docs across 3 repos and do it in one place, so there is still arguments to keep most of the important stuff here.